data_processing_module3.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+from sklearn import preprocessing
+from sklearn.cross_validation import train_test_split
+import random
+
+X1=np.asarray([random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10)], dtype=np.float)
+X2=np.asarray([random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10)], dtype=np.float)
+X3=np.asarray([random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10)], dtype=np.float)
+Y=np.asarray([random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10)], dtype=np.float)
+W=np.asarray([random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10), random.sample(xrange(1000), 10)], dtype=np.float)
+
+
+def data_processing(X1, X2, X3, Y, W):
```

Comment:
It'd be best to give your matrices more informative names, e.g. `X_jets` instead of `X1`